bears/python/Flake8Bear.py

```diff
@@ -0,0 +1,74 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='flake8',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+):(?P<column>\d+):\s'
+                     r'(?P<severity>[EFW]|C9|N8)[0-9]{2,3}\s'
+                     r'(?P<message>.*)',
+        severity_map={
+            "F": RESULT_SEVERITY.MAJOR,
+            "E": RESULT_SEVERITY.MAJOR,
+            "W": RESULT_SEVERITY.NORMAL,
+            "N8": RESULT_SEVERITY.INFO,
+            "C9": RESULT_SEVERITY.INFO})
+class Flake8Bear:
+    """
+    Checks for errors based on pyflakes, pycodestyle and mccabe.
+
+    See <https://gitlab.com/pycqa/flake8> for more information
+    """
+    LANGUAGES = {"Python2, Python 3, PyPy"}
+    REQUIREMENTS = {PipRequirement('flake8', '2.*')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/06mjx6o5idamjggv71gr5n1o2'
+    CAN_DETECT = {'Formatting', 'Unused code', 'Complexity',
+                  'Undefined Element', 'Variable Misuse'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         max_line_length: int=79,
+                         flake8_select: typed_list(str)=(),
+                         flake8_ignore: typed_list(str)=(),
+                         disablenoqa_config: bool=True,
+                         doctests_config: bool=True):
+        """
+        :param max_line_length:
+            Maximum number of characters on a line
+        :param flake8_select:
+            List of checked errors by specifying which errors to check for.
+            Can't be used together with ``flake8_ignore``.
+        :param flake8_ignore:
+            List of checked errors by specifying which errors to ignore. Can't
+            be used together with ``flake8_select``.
+        :param disablenoqa_config:
+            Set to true to enable flake8 to report errors on the same line as
+            a "# NOQA" comment.
+        :param doctests_config:
```

Comment:
The `_config` is redundant.  All of these parameters are config.
this should be something like `check_doctest_syntax`